indico/MaKaC/PDFinterface/conference.py

```diff
@@ -78,6 +77,30 @@
 ]
 
 
+# Change reportlab default pdf font Helvetica to indico ttf font,
+# because of better support for international characters.
+# Default font can't be easily changed by setting new value to
+# reportlab.rl_config.canvas_basefontname, because this variable
+# is used directly to initialize state of several reportlab modules
+# during their (first) import.
+def getSansStyleSheet():
```

Comment:
@vokac yes, indeed, but anyway you are calling it yourself, so, you can call it whatever you want at the end of the day. It may seem pedantic, but we really prefer to have all patches in snake case, unless it's a method you have to override.